### DISCUSSION TITLE
Set default (CUSTOM) error messages for all zod Instances.

### DISCUSSION THREAD

**Discussion by davidreyg** [colinhacks/zod#977] [Q&A]
How Can I set Custom error messages or use LOCALES like in _yup-locales_...  using zod?
And every instance of zod use them when throwing an error..

**Comment by JacobWeisenburger** [comment]
Could you provide a code example of what you are wanting? I'm having a hard time understanding the desired result.

**Comment by willd-mwp** [comment]
@davidreyg 

This worked for me https://github.com/colinhacks/zod/blob/master/ERROR_HANDLING.md#customizing-errors-with-zoderrormap
